### PR TITLE
readme and package.json fix

### PR DESCRIPTION
**Pull request by nicofuccella**
add readme
updates package.json

close issue #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,140 @@
...
+### Running the tests
+
+You can run tests running:
+```shell
+$ npm run test
+```
+
+### Publishing
+
+In order to publish a new version of the package on npm you must first create a git tag with the current content of the package:
```
Comment (giowe) [inline README.md]: useless, remove this section

File: README.md [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+  noColors: false // enable the console colors of the debug logs
+})
+
+container
+  .register(mysqlComponent)
+  .register(productsComponent)
+  .init()
+```
+
+### Component definition
```
Comment (giowe) [inline README.md:43]: consider  using mysql-component class as example

File: README.md (hunk 1 above)
Comment (nicofuccella) [reply README.md]: Fine

File: README.md (hunk 2 above)
Comment (nicofuccella) [reply README.md:43]: I'll add a new section `Examples`

**Pull request merged**